README.rst

```diff
@@ -43,6 +43,49 @@ __ `OpenBSD package`_
 __ `openSUSE package`_
 __ `Ubuntu package`_
 
+Docker
+------
+
+A docker package exists for this project::
+
+  $ git pull ghcr.io/rss2email/rss2email:master
+
+Then it can be used like::
+
+  $ docker run  -v ./data:/data -v ./config:/config rss2email list
+  
+You need to create the config file and have some kind of SMTP server set up in the config file. With ``--net host`` option you can use SMTP server on the host.
+
+Docker compose
+~~~~~~~~~~~~~~
+
+Here is an example docker-compose.yaml::
+
+  version: '2'
+  services:
+    rss2email:
+      image: ghcr.io/rss2email/rss2email:master
+      volumes:
+        - ./config:/config
+        - ./data:/data
+      entrypoint: "/venv/bin/python3 -c 'import threading; threading.Event().wait()'"
+      labels:
+        chadburn.enabled: "true"
+        chadburn.job-exec.rss2email.no-overlap: "true"
+        chadburn.job-exec.rss2email.schedule: "@every 5m"
+        chadburn.job-exec.rss2email.command: "r2e run"
+
+The example docker-compose snippet sets up chadburn (or ofelia) job scheduler to run r2e periodically. 
+
+Some background for entrypoint override: For the labels to be picked up, it is required that the container keeps running which is ensured with the entrypoint. Python's Event().wait() was used as there is no shell and this method should use less cpu than sleeping or looping (with sleep). You can of course abandon this and just call r2e from any other job scheduler as this entrypoint only circumvents chadburn/ofelia limitations.
```

Comment:
This seems like too much detail for something that is outside of r2e. In future iterations of chadburn this workaround will be made redundant, but it will remain here until someone cares enough to update. Also, it seems a little excessive for something to be kept running indefinitely just to wake up a couple of times per day. The CPU usage while running may be close to 0%, but RAM is also a resource, and some cloud platforms might charge you for every second of run time.

Reply:
The CPU usage here should be lower than any sleep command available.

I don't think it's possible for Chadburn or Ofelia to do exec on a stopped service, though other job schedulers operating outside of Docker might be able to.

I originally had this set up to run via docker run in cron, but opted to go for a cleaner docker-centric setup where I have everything in one place.

Reply:
In future iterations of chadburn this workaround will be made redundant - is this related to the job-run functionality? I was not aware of that coming :)

Reply:
@amiryal In future iterations of chadburn this workaround will be made redundant - is this related to the job-run functionality? I was not aware of that coming :)